.github/workflows/publishing.yml

```diff
@@ -713,23 +719,50 @@ jobs:
           name: x86_64-installer
           path: /tmp/install
 
-      - name: Install runtime dependencies
+      - name: Runtime dependencies (apt)
+        if: startsWith(matrix.os_image, 'ubuntu')
+        run: |
+          apt-get update && apt-get install -y --no-install-recommends \
+            wget ca-certificates libstdc++-11-dev libopenmpi-dev
+          distro=`echo ${{ matrix.os_image }} | tr -d . | tr -d :`
+          CUDA_DOWNLOAD_URL=https://developer.download.nvidia.com/compute/cuda/repos
+          wget "${CUDA_DOWNLOAD_URL}/$distro/x86_64/cuda-keyring_1.1-1_all.deb"
+          dpkg -i cuda-keyring_1.1-1_all.deb && version_suffix=11-8
+          apt-get update && apt-get install -y --no-install-recommends \
+              cuda-nvtx-${version_suffix} cuda-cudart-${version_suffix} \
```

Comment:
Just a suggestion: I'm not sure that `cuda-nvtx` is really needed. Since it is a big download, it may speed it up to remove it, but I'm ok either way.

Reply:
I think you are right and this was originally needed for the tensornet simulator.

Reply:
For the record: cublas is the only cuda runtime dependency for the nvidia-mgpu backend. cublas, cusolver and cudart are needed for the tensornet and tensornet-mps backends. Maybe we should just link these statically in the installer.